pdns/pdnsutil.cc

```diff
@@ -401,6 +405,39 @@ static int checkZone(DNSSECKeeper &dk, UeberBackend &B, const DNSName& zone, con
       continue;
     }
 
+    if (rr.qtype.getCode() == QType::SVCB || rr.qtype.getCode() == QType::HTTPS) {
+      vector<string> parts;
+      stringtok(parts, rr.content);
+      if (std::atoi(parts.at(0).c_str()) == 0 && parts.size() > 2) {
```

Comment:
Wouldn't `parts.at(0)` throw if `rr.content` is empty, or contains only white spaces?

Reply:
It shoudl have already error'd above (in the parsing step). But perhaps we need to eb more careful indeed.

Reply:
If we are not supposed to reach that part of the code with an invalid content, I'm fine with an exception being raised. :-)

Reply:
fwiw, on [line 399](https://github.com/PowerDNS/pdns/pull/9369/files#diff-e69997cee73802012aa0260286fbd5c7R399) we `continue` if marshalling fails.